frontend/src/home/components/NextStep.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from "react";
+
+import SidebarSection from "./SidebarSection.jsx";
+
+export default class NextStep extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            next: null
+        };
+    }
+
+    async componentWillMount() {
+        let response = await fetch("/api/setup/admin_checklist", { credentials: 'same-origin' });
```

Comment:
@tlrobinson If I recall you had a reason for holding off "redux-ifying" this particular component but I can't seem to remember what it was.